### ISSUE TITLE
Fix satsback event for it to be an internal transaction

### ISSUE THREAD

**Issue by rapax00** [lawalletio/satsback#4]
The *internal transactions* of LaWallet have a tag with metadata encrypted to make it an internal transaction.

I need help creating this metadata and including it in the event.

The event is created [here](https://github.com/lawalletio/satsback/blob/1152b81982461c660eb60824422a4a91b6f6ab41/src/utils/makeEvent.ts#L95).

**Comment by rapax00** [comment]
@cuervoshi you know about this! hehe 🐦‍⬛

**Comment by rapax00** [comment]
# Implemented

The crow told me about it.

# How to create metadata

Metadata is a tag of a Nostr event like this:

```json
[ "metadata", <string, true or false if it is encrypted>, <string, encrypted type>, <string, encrypted content> ]
```

Content:

```json
{ 
  "sender": <string, walias/lud16>,
  "receiver": <string, walias/lud16>
}
```

# My implementation

- The sender is the module, and the receiver is in the payment event.
- The encrypted type is `nip04` of `nostr-tools`

These changes are merged in [PR #6](https://github.com/lawalletio/satsback/pull/6) and can be found in [this line](https://github.com/lawalletio/satsback/commit/c39ebd1abff594b01fc2e03b2f39719139955b63#diff-fcba3e0438b6e8c45bff621bad05aec1a903ee50142e9c5fd277558950029233R98).
